src/catalog/pgduckdb_transaction.cpp

```diff
@@ -82,26 +82,32 @@ SchemaItems::GetTable(const string &entry_name) {
 	info.table = entry_name;
 	Cardinality cardinality = 1;
 	if (!PostgresTable::SetTableInfo(info, rel)) {
+		RelationClose(rel);
```

Comment:
Here too we would leak `rel` when returning

Reply:
Afaict that function can (surprisingly) only return true. So I guess this was not the cause.

Reply:
Yeah it wasn't the root cause indeed. And I didn't look at `!PostgresTable::SetTableInfo` if it only returns true, then I'll remove the branch :-)

Reply:
> if it only returns true, then I'll remove the branch :-)

Sounds good let's make it a `void` function indeed.

Reply:
Yep, exactly what I did :-)